ParcelDistributionCenter/ParcelDistributionCenter.Concole/Program.cs

```diff
@@ -11,7 +11,7 @@ static void Main(string[] args)
 
             // string ParcelUser = File.ReadAllText(@"D:\packages.json");//odczytanie json z pliku. Ścieżkę dodana na sztywno.
             //plik Json umieszczony w projekcie :jcszr8-DeBurgerzy\ParcelDistributionCenter\ParcelDistributionCenter.Concole\bin\Debug\net6.0
-            string ParcelUser = File.ReadAllText("packages.json");
+            string ParcelUser = File.ReadAllText("..\\..\\..\\..\\ParcelDistributionCenter.Model\\json\\packages.json");
             List<Parcel> UserList1 = JsonConvert.DeserializeObject<List<Parcel>>(ParcelUser);//wpisanie json do listy
```

Comment:
Bardzo fajna metoda. Pewnie jak zrobimy model całości to trzeba będzie ją przenieść gdzieś z głównego pliku Program.